go/nbs/store.go

```diff
@@ -397,6 +397,10 @@ func (nbs *NomsBlockStore) updateManifest(current, last hash.Hash) error {
 		nbs.mt = nil
 	}
 
+	if current == last && len(nbs.tables.novel) == 0 {
```

Comment:
oh, I thought you meant that we'd do this _after_ we go get the updated manifest. I think we have to do that, don't we?